src/openqaoa-core/algorithms/qaoa/qaoa_workflow.py

```diff
@@ -13,6 +15,97 @@
 from ...optimizers.qaoa_optimizer import get_optimizer
 
 
+def circuit_routing(
+    device: DeviceBase,
+    problem_to_solve: List[List[int]],
```

Comment:
This is the list of two qubits gates, right? Let's give the variable a better name then!